Neptune/IMessage.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Neptune
+{
+
+    public interface IRequest<IS>
```

Comment:
Are you all going to require anything like StyleCop for standardizing documentation? And general formatting of code.

Reply:
Yes absolutely. I've run some sane code formatting fixes built into vs2019 until we settle on a standard.